### ISSUE TITLE
Blank output generated

### ISSUE THREAD

**Issue by jean** [peterhellberg/hiro#3]
I use the `api.apib` file as exported from apiary.com
The generated file looks like it contains only scaffolding. 
No error is shown. 

I get the same result when I give iglo's demo as input.

```
~/local/go$ ./bin/hiro -input=/home/john/local/go/src/github.com/subosito/iglo/examples/API.md -output=output.html 
```

If I use iglo directly (after hacking `main.go` to point at the my apib file), I do get the correct output:

```
~/local/go/src/github.com/subosito/iglo/examples/api-exporter$  go run main.go 
```

**Comment by behe** [comment]
You need to update to v1.0 of Drafter:

```
brew reinstall --HEAD \
https://raw.githubusercontent.com/apiaryio/drafter/v1.0.0/tools/homebrew/drafter.rb
```

**Comment by jean** [comment]
@behe should later versions work?

```
$ drafter --version
v2.3.0-pre.0
```

[output.html.txt](https://github.com/peterhellberg/hiro/files/198970/output.html.txt)

**Comment by behe** [comment]
Excellent question. Have you tried with a later version @peterhellberg?

**Comment by peterhellberg** [comment]
No, [iglo](https://github.com/peterhellberg/iglo) (which hiro uses to actually convert the drafter output) requires `v.1.x.x` of drafter for now.

I might get around to fix this, but don’t hold your breath for now.

**Comment by peterhellberg** [comment]
It looks like the upstream version of iglo now requires (at least) drafter `2.0.1`. This probably means that I can remove my fork when fixing this issue.

**Comment by peterhellberg** [comment]
And after some digging it looks like the current drafter (v2.3.0-pre.2) is horribly broken since the output when using `--format json` isn’t _actually_ valid JSON. **sigh**

**Comment by peterhellberg** [comment]
I’ve now released version 0.0.2 of the hiro command line tool, it is once again using github.com/subosito/iglo instead of my own (now deleted) fork of iglo. I’ve decided to require **drafter v2.3.x** to be installer in order to use this tool. (So no backwards compatibility with older drafter versions)
